recipes/gpi_core/run_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash +x
+if [ "$(uname)" == "Darwin" ]; then
+    OUTPUT=`gpi --nogui repo_contents/testNets/TestNetwork.net | grep "gpi.canvasGraph:384"`
+
+echo "$OUTPUT"
```

Comment:
Can you indent the rest of this block (at first I didn't realise this was Darwin only)

Reply:
Will do, sorry for the ugly formatting there!